contracts/rewards/VestingRecipient.sol

```diff
@@ -185,14 +195,23 @@ contract VestingRecipient is Initializable {
     // Internal
     // ----------------------------------------------------------
 
+    /**
+     * @dev Locks reward tokens on the aura locker.
+     * @param _amount Amount of reward tokens to  lock.
+     */
     function _lock(uint256 _amount) internal {
-        require(address(auraLocker) != address(0), "!auraLocker");
         IERC20(rewardToken).safeApprove(address(auraLocker), _amount);
         IAuraLocker(auraLocker).lock(address(this), _amount);
     }
 
-    function _withdrawable() internal view returns (uint256) {
+    /**
+     * @dev Upbound amount reward tokens to withdraw.
+     */
+    function _maxWithdrawable() internal view returns (uint256) {
```

Comment:
I’m thinking we actually need to tweak this to account because you will get AURA rewards that you wouldn’t be able to withdraw. Rewards from being vlAURA holder you know.